build.sbt

```diff
@@ -57,9 +57,9 @@ val AkkaPersistenceCassandraVersion = "0.102"
 val PrometheusClientVersion = "0.6.0"
 val ScalaTestVersion = "3.0.5"
 val ProtobufVersion = "3.9.0"
-val GraalVersion = "19.3.0"
-val DockerBaseImageVersion = "openjdk:8-jre-slim-buster"
-val svmGroupId = if (GraalVersion startsWith "19.2") "com.oracle.substratevm" else "org.graalvm.nativeimage"
+val GraalVersion = "20.0.0"
+
+val DockerBaseImageVersion = "adoptopenjdk/openjdk8:debian"
```

Comment:
@srikanthops This image passes the tests, and should be based on Buster. Can you verify?